nautobot/docs/plugins/development.md

```diff
@@ -574,6 +574,12 @@ A `PluginMenuButton` has the following attributes:
 !!! note
     Any buttons associated within a menu item will be shown only if the user has permission to view the link, regardless of what permissions are set on the buttons.
```

Comment:
If this note is correct (I believe it is) it should probably be included in `navigation-menu.md`.